<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Promotion</class>
 <widget class="QWidget" name="Promotion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1900</width>
     <height>1000</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Desktop/ADMIN/Promotion.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="AfficherP">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>1260</width>
     <height>630</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <widget class="QWidget" name="Ajout">
    <attribute name="title">
     <string>Ajouter Promotion</string>
    </attribute>
    <widget class="QLineEdit" name="Taux">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>160</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Ltaux">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Taux</string>
     </property>
    </widget>
    <widget class="QLabel" name="Lcode">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Code</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="Date_Fin">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>120</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="LdateF">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>110</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Date Fin</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Code">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>480</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Desktop/ADMIN/BoutonAjouter.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="Ajouter">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>490</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Afficherr">
    <attribute name="title">
     <string>Afficher Promotion</string>
    </attribute>
    <widget class="QTableWidget" name="Afficher">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>531</width>
       <height>471</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans Pro</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="autoScrollMargin">
      <number>20</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>130</number>
     </attribute>
     <column>
      <property name="text">
       <string>Code</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="background">
       <color alpha="0">
        <red>0</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date Fin</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Taux</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date Debut</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
    </widget>
    <widget class="QLineEdit" name="modf_code">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>90</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>180</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Taux</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>260</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Date Fin</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="modf_taux">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>180</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>100</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Code</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="modf_datef">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>260</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>23</hour>
       <minute>0</minute>
       <second>0</second>
       <year>1900</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="minimumDateTime">
      <datetime>
       <hour>21</hour>
       <minute>0</minute>
       <second>0</second>
       <year>1900</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="timeSpec">
      <enum>Qt::UTC</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>1050</x>
       <y>250</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Desktop/ADMIN/BoutonSupprimer.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>1050</x>
       <y>170</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Desktop/ADMIN/BoutonModifier.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="Enregistrer">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>170</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Supprimer">
     <property name="geometry">
      <rect>
       <x>1050</x>
       <y>250</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="Abonne">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Abonnement">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Cours">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Stock">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Back">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>910</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
